### PR TITLE
blog: Add blog post for GSoC'23: Binary Compatibility

### PR DESCRIPTION
**Pull request by i-Pear**

### CODE REVIEW

File: content/en/blog/2023-06-23-unikraft-gsoc-app-compat-1.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Currently, we have successfully generated a preload library for the libc wrapper targeting system calls.
+Disregarding TLS considerations, this approach has allowed the Redis Benchmark with dynamic libraries, to achieve performance close to that of a native image.
+
+B. VDSO:
+TBD.
+
+C. CI/CD:
+The regression tests can now be executed on GitHub Actions.
+
+## Next steps
```
Comment (razvand) [inline content/en/blog/2023-06-23-unikraft-gsoc-app-compat-1.md:52]:
```suggestion
## Next Steps
```

File: content/en/blog/2023-06-23-unikraft-gsoc-app-compat-1.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- Rapidly implementing a VDSO library to accelerate time-related system calls.
+
+C. CI/CD:
+
+- Find a server to host the necessary data for CI/CD and officially incorporate CI/CD into Unikraft's workflow.
+
+## About Me
+
+I'm Tianyi Liu, a first-year graduate student at Nanjing University.
```
Comment (razvand) [inline content/en/blog/2023-06-23-unikraft-gsoc-app-compat-1.md:74]: Add links to your GitHub page and the university page.

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @i-Pear, this looks great.

Please use "Add blog post" instead of "Add a blog" in the commit message. Also add a short summary paragraph detailing the blog post contents.

See the linter errors.

Generally look into the [contribution guidelines](https://unikraft.org/docs/contributing/submitting-changes/) and also see the [documentation on documentation](https://unikraft.org/docs/contributing/docs/).

**Comment by i-Pear** [comment]
@razvand Thanks for you review, I have completed all the revisions.

**Review by razvand — CHANGES_REQUESTED** [review]
Hi, @i-Pear, please use your full name both for the commit author and for the signed off message.

Also add a paragraph description of the commit message. Something like:

"""
Part of the "Extending Binary Compatibility" GSoC'23 project, this blog post details work in first 3 weeks of GSoC, together with planned work for the upcoming 3 weeks.
"""

See my other comments inline.
```

**Pull request merged**